xinference/deploy/cmdline.py

```diff
@@ -84,12 +84,39 @@ def get_endpoint(endpoint: Optional[str]) -> str:
         return endpoint
 
 
-@click.group(invoke_without_command=True, name="xinference")
+@click.group(
+    invoke_without_command=True,
+    name="xinference",
+    help="Xinference command-line interface for serving and deploying models.",
+)
 @click.pass_context
-@click.version_option(__version__, "--version", "-v")
-@click.option("--log-level", default="INFO", type=str)
-@click.option("--host", "-H", default=XINFERENCE_DEFAULT_LOCAL_HOST, type=str)
-@click.option("--port", "-p", default=XINFERENCE_DEFAULT_ENDPOINT_PORT, type=int)
+@click.version_option(
+    __version__,
+    "--version",
+    "-v",
+    help="Show the current version of the Xinference tool.",
+)
+@click.option(
+    "--log-level",
+    default="INFO",
+    type=str,
+    help="""Set the logger level. Options listed from most log to least log are:
+              NOTSET > DEBUG > INFO > WARNING > ERROR > CRITICAL (Default level is INFO)""",
```

Comment:
```suggestion
             DEBUG > INFO > WARNING > ERROR > CRITICAL (Default level is INFO)""",
```

NOTSET is not an effective level. See: https://docs.python.org/3/library/logging.html#logging.Logger.setLevel